android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UriBeaconDiscoveryService.java

```diff
@@ -270,19 +274,28 @@ public void onUrlMetadataIconReceived() {
 
   @Override
   public void onMdnsUrlFound(String url) {
-    if (!mUrlToUrlMetadata.containsKey(url)) {
-      // Fabricate the device values so that we can show these ersatz beacons
-      String mockAddress = generateMockBluetoothAddress(url.hashCode());
-      int mockRssi = 0;
-      int mockTxPower = 0;
-      mUrlToUrlMetadata.put(url, null);
-      mDeviceAddressToUrl.put(mockAddress, url);
-      mRegionResolver.onUpdate(mockAddress, mockRssi, mockTxPower);
-      MetadataResolver.findUrlMetadata(this, UriBeaconDiscoveryService.this, url, mockTxPower, mockRssi);
-    }
+    onLanUrlFound(url);
   }
 
-  private void startSearchingForUriBeacons() {
+    @Override
+    public void onSsdpUrlFound(String url) {
```

Comment:
Please align the indentation of onSsdpUrlFound, onLanUrlFound, and startSearchingForUriBeacons to match the surrounding methods.